### ISSUE TITLE
Tooltip - Global limited tooltip ingore the for="xxxx" and shows for every html tag with an title param a tooltip

### ISSUE THREAD

**Issue by bzebedin** [primefaces-extensions/primefaces-extensions.github.com#289]
You can also see it in the showcase.
http://www.primefaces.org/showcase-ext/sections/tooltip/globalLimitedTooltips.jsf

The showcase says that only tooltips for @(.ui-state-error) shall be shown.
If you have a detail look on the showcase page you can see that different tooltips are shown.
e.g. if you hover the primefaces extensions logo at top left a tooltip is shown.
the same if you hover attribute names in the documentation section (bottom right)
.... for every name a tooltip will be shown.

I'm fighting with this problem in my application because for every icon-button we have (with title attribute) a tooltip is shown. But we only want to use this global tooltip for error-handling.

In my opinion this is not a correct behaviour of a global limited tooltip.

kind regards
Bernhard

**Comment by melloware** [comment]
I have debugged and fixed this issue.   

While I was in here I have also done the following:

1. Updated to qTip2 3.0.3 from 2.0.0
2. Added new property header="XXX" which will display a header in the tooltip
3. Added new property styleClass="" allowing you to override the ThemeRoller theme with your own or built in QTip themes.

I have updated the showcase and all examples to show this is fixed as well as demonstrate the new properties added.
